js/src/utils/spawn.js

```diff
@@ -24,7 +26,10 @@ exports.spawnNodes = spawnNodes
 
 // Spawn n nodes, getting their id's and setting them as `peerId` on the nodes
 function spawnNodesWithId (n, factory, callback) {
-  timesSeries(n, (_, cb) => spawnNodeWithId(factory, cb), callback)
+  spawnNodes(n, factory, (err, nodes) => {
+    if (err) return callback(err)
+    map(nodes, identify, callback)
+  })
```

Comment:
Apologies the change in this file should have been proposed a separate PR - this simply ensures the ID calls are done in parallel which will speed up the tests a little.